artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/broker/AMQPLargeMessage.java

```diff
@@ -622,7 +624,15 @@ public long getPersistentSize() throws ActiveMQException {
 
    @Override
    public void reencode() {
+      reEncoded = true;
+   }
+
+   public void setReEncoded(boolean reEncoded) {
+      this.reEncoded = reEncoded;
```

Comment:
Nit. Why is the E capitalised. Reencoded is a single word. https://en.m.wiktionary.org/wiki/reencoded

Reply:
right.. fixed it. thanks